.github/workflows/build-wasm_of_ocaml.yml

```diff
@@ -33,6 +33,11 @@ jobs:
             separate_compilation: true
             jane_street_tests: false
             all_jane_street_tests: false
+          - os: windows-latest
+            ocaml-compiler: "5.2"
+            separate_compilation: true
+            jane_street_tests: true
```

Comment:
@vouillon, `jane_street_tests` currently doesn't change whether we perform more and less tests. Is that expected ?